manifests/service_limits.pp

```diff
@@ -33,7 +33,7 @@
   Enum['present', 'absent', 'file'] $ensure                  = 'present',
   Stdlib::Absolutepath              $path                    = '/etc/systemd/system',
   Boolean                           $selinux_ignore_defaults = false,
-  Optional[Systemd::ServiceLimits]  $limits                  = undef,
+  Optional[Systemd::Servicelimits]  $limits                  = undef,
```

Comment:
I think these are case insensitive, but is there a reason to change it? Seems like it'd be better to leave it matching the case of the type definition.

Reply:
Yep - I removed it originally and tried using Systemd::Unit::Service there.